Failure is classloading dependant, so the failure one has to be run alone:

```
sbt test:testOnly FailureSpec
```

